crates/runtime/src/instance.rs

```diff
@@ -516,6 +513,14 @@ impl Instance {
             .unwrap_or_else(|| panic!("no table for index {}", table_index.index()))
             .set(index, val)
     }
+
+    fn alloc_layout(&self) -> Layout {
+        let size = mem::size_of_val(self)
+            .checked_add(usize::try_from(self.offsets.size_of_vmctx()).unwrap())
```

Comment:
I wonder if `size_of_vmctx` can ever return a different value?

Reply:
It can, yeah, it's dynamically sized based on the contents of the wasm module (e.g. number of globals/tables/etc)

Reply:
Oh, yeah, I meant during the lifetime of a module. I think it shouldn't, right? Otherwise, we are risking with deallocating the value providing a different layout.

Reply:
Ah good point, and yeah it's safe to assume that this doesn't change for the lifetime of the instance since it's computed from the module. This could likely be stored in the `Arc` allocation to show that it doesn't change, but we can tackle that later.